src/app/pages/reports-dashboard/reports.service.ts

```diff
@@ -162,4 +167,18 @@ export class ReportsService {
       }),
     );
   }
+
+  openNetdata(password: string): void {
+    // Thats the only way I could manage to have the browser save the password
+    // to re-use when the new tab is opened.
+    // HttpClient seems to do it through HttpHeaders which won't cache in browser.
+    const xhr = new XMLHttpRequest();
+    this.loggedInUser$.subscribe((user) => {
+      xhr.open('GET', '/netdata/', true, user.pw_name, password);
```

Comment:
It looks like doing this just puts username and password in the url: 
`http://root:password@localhost:4200/netdata/`.
If we want to do this (security concerns?), maybe you can form the URL yourself and use HttpClient (it will be easier in tests).